docs/spec.md

```diff
@@ -6138,6 +6203,14 @@ def dequantize_op_quantize(op, *inputs_and_output_type):
   float_result = op(*float_inputs)
   rounded_result = round_nearest_even(float_result / scale(output_type))
   return clamp(storage_min(output_type), rounded_result, storage_max(output_type))
+
+def dequantize_op_quantize_select(op, pred, *inputs_and_output_type):
```

Comment:
If we're specializing dequantize_op_quantize to a specific op, let's go all the way:

```python
def dequantize_select_quantize(pred, on_true, on_false):
  float_on_true = (on_true - zero_point(on_true)) * scale(on_true)
  float_on_false = (on_false - zero_point(on_false)) * scale(on_false)
  float_result = select(pred, float_on_true, float_on_false)
  rounded_result = round_nearest_even(float_result / scale(output_type))
  return clamp(storage_min(output_type), rounded_result, storage_max(output_type))
```

Reply:
done